transport-udt/pom.xml

```diff
@@ -78,7 +82,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <forkMode>always</forkMode>
+          <reuseForks>false</reuseForks>
```

Comment:
Is this just to replace "deprecated" naming ?

Reply:
yes